ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -1189,6 +1189,8 @@ paths:
       operationId: get-transport-document
       description: |
         Retrieves the `Transport Document` with the `transportDocumentReference` in the path.
+
+        **Condition:** Once the `Transport Document` has been Issued (`transportDocumentStatus='ISSUED'`) - the order of **ALL** lists/arrays in this payload **MUST** be preserved by provider of the API.
```

Comment:
can we add as by the provider of the API? Applicable to all the places this condition is added